html/pfappserver/root/static.alt/src/views/Reports/_components/ReportTable.vue

```diff
@@ -33,47 +42,43 @@
           </b-container>
         </b-col>
       </b-row>
-      <b-table :items="items" :fields="visibleColumns" :sort-by="sortBy" :sort-desc="sortDesc" :sort-compare="sortCompare" v-model="tableValues"
-        @sort-changed="onSortingChanged" responsive hover></b-table>
+      <b-table stacked="sm" :items="items" :fields="visibleColumns" :per-page="pageSizeLimit" :current-page="requestPage" :sort-by="sortBy" :sort-desc="sortDesc" :sort-compare="sortCompare"
```

Comment:
You can't use both the `stacked` and `responsive` attributes at the same time. Keep `responsive`, remove `stacked`.